lib/AggregateGraph.py

```diff
@@ -61,13 +61,13 @@ def createAggregateGraph(log_directory, channel_name, output_directory, starting
 			filePath=log_directory+temp1+str(folderiterator)+"/"+temp2+str(fileiterator)+"/"+channel_name+".txt"   
 			if not os.path.exists(filePath):
 				if not((folderiterator==2 and (fileiterator ==29 or fileiterator ==30 or fileiterator ==31)) or ((folderiterator==4 or folderiterator==6 or folderiterator==9 or folderiterator==11) and fileiterator==31 )): 
-					print "[Error] Path "+filePath+" doesn't exist"
+					print ("[Error] Path "+filePath+" doesn't exist")
```

Comment:
ok. i ran it using Parser.py using python3 and it asked explicitly for the Parenthesis. should i remove this?